### DISCUSSION TITLE
Tags Personalizadas e competividade

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#975] [Ideia]
### Servidor afetado

Todos

### Qual o propósito da ideia?

Criar uma competividade e incentivo de farmar mc

### Descrição da ideia

Tem tags de torneios com destaque e que são bem diferentes das demais no tab. Ideia é começar manualmente entregar e a cada dia primeiro do mês checar se a pessoa ainda merece permanecer com a tag.

### Aplicação

Primeiramente começar com entrega manual da tag pois não possui nada programado pra ser automático. Como não é obrigatório atualmente, podemos ter esse esforço e eu fico responsável com a tarefa.

Recompensas :
- Tag Personalizada de duração mensal
- 1.5k de cash para o top 1

### Referências

Sugestão de EricRocha

**Comment by leogianfagna** [comment]
Já está sendo produzido em #848
